circle.yml

```diff
@@ -126,4 +124,8 @@ test:
       unicode
       unicode/utf16
       unicode/utf8
+    - >
+      gopherjs test --short --minify --run='^Test'
+      net/http/cookiejar
+      reflect
```

Comment:
Well, it would allow you to simplify circle.yml back to the original version. I think it's worth doing, if it's indeed possible. Thanks!